test/cypress/src/client.js

```diff
@@ -0,0 +1,34 @@
+import {
+    Client,
+    AccountId,
+    PrivateKey,
+    Hbar,
+    AccountCreateTransaction,
+} from "@hashgraph/sdk";
+
+const operatorId = AccountId.fromString('0.0.2');
+const operatorKey = PrivateKey.fromString('302e020100300506032b65700422042091132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137');
+const client = Client.forNetwork({"http://127.0.0.1:50213":"0.0.3"}).setOperator(operatorId, operatorKey);
```

Comment:
nit: we can maybe get those from .env

Reply:
I thought about that, but the AccountId and private key should always be the same, probably the proxy address could be subject to change when we implement multiple consensus nodes. I suggest we leave it as it is for now.

Reply:
I thought about that, but the AccountId and private key don't need to be configurable. The proxy address may be subject to change when we implment multiple consensus nodes. I think we can leave it as is for now.